content/docs/0.7.x/troubleshooting/index.md

```diff
@@ -313,3 +313,70 @@ initContainers:
 ```
 
 </p></details>
+
+## Fully Qualified Domain Names cannot be reached from within the cluster
+
+Depending on your Kubernetes cluster configuration, certain hostnames cannot be reached from within the Kubernetes cluster. This is usually visible via an error message that looks as follows:
+```
+Failed to send cloudevent:, Post http://event-broker.keptn.svc.cluster.local/keptn: dial tcp: lookup event-broker.keptn.svc.cluster.local: Try again
+```
+
+The problem can appear in virtually any service and scenario:
+
+1. LoadGenerator for keptn/examples
+1. Prometheus-Service/Prometheus-SLI-Service trying to access prometheus
+1. Dynatrace-Service/Dynatrace-SLI-Service trying to access a Dynatrace environment
+1. Unleash-Service trying to access unleash
+1. Any keptn-service trying to send a CloudEvent via the event-broker
+
+<details><summary>Expand instructions</summary>
+<p>
+
+
+**Problem**: Trying to access certain hostnames does not work within the cluster.
+
+The reason behind this is that some Kubernetes cluster configurations have issues when it comes to resolving internal hostnames like `service.namespace.svc.cluster.local`, but potentially reaching ANY hostname might fail, e.g., trying to fetch a URL via `wget keptn.sh`.
+
+**Analysis**: To find out whether you are affected or not, please run an `alpine:3.11` container that tries to access the Kubernetes API or any external hostname, e.g.:
+
+```
+kubectl run -i --restart=Never --rm test-${RANDOM} --image=alpine:3.11 -- sh -c "wget --no-check-certificate https://kubernetes.default.svc.cluster.local/api/v1"
+```
+
+```
+kubectl run -i --restart=Never --rm test-${RANDOM} --image=alpine:3.11 -- sh -c "wget https://keptn.sh"
+```
+
+If in any of the above instances you get a "bad address", then you might be affected, e.g.:
```

Comment:
Change `might` to `are`: `If in any of the above instances you get a "bad address", then you are affected,`